### ISSUE TITLE
Don't override the password character

### ISSUE THREAD

**Issue by hadess** [jangernert/FeedReader#205]
Right now I see asterisks instead of filled in circles, like every other GTK+ app.

**Comment by jangernert** [comment]
How about suggesting ideas as such and not sounding like you are demanding things to be done for you...
I don't get paid for this, so I can at least expect you to hit a nicer tone if you want me to work for free.

Maybe I am reading this wrong, but this is the picture in my head while reading this (and the other report) =/

**Comment by ZanderBrown** [comment]
https://github.com/jangernert/FeedReader/blob/0e7503fa11ef244ff8fae88ac37ecd89d7dc3254/plugins/backend/owncloud/OwncloudNewsLoginWidget.vala#L59 you set * to display instead of the entered text

Line 60 of the same file should be sufficient combined with `m_passwordEntry.input_purpose = Gtk.InputPurpose.PASSWORD`

**tl;dr**

replace:
```
m_passwordEntry.set_invisible_char('*');
m_passwordEntry.set_visibility(false);
```
with:
```
m_passwordEntry.input_purpose = Gtk.InputPurpose.PASSWORD;
m_passwordEntry.set_visibility(false);
```

**Comment by jangernert** [comment]
I know that this one should not be complicated. I just didn't like the attitude.

**Comment by hadess** [comment]
@jangernert I file and fix (GNOME) bugs for a living, so I usually file bugs in the imperative form, especially for design bugs.

**Comment by hadess** [comment]
Don't try to find attitude when there is none. I hope that FeedReader is the tool that I get to use (once I can figure out why tt-rss doesn't work on my NAS). Will you reopen #204?

**Comment by jangernert** [comment]
That's the difference, you get money for it, I don't. So if something sounds rude to me, my desire to spend my free-time working on what person X on the internet told me to, is suddenly much lower.

**Comment by afranke** [comment]
Money doesn't come into account here. It is common to use the imperative form, because it's clearer and more efficient. Don't take this personnally.
